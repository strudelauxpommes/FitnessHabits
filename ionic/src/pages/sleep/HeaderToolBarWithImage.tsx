import { IonBackButton, IonButtons, IonIcon, IonTitle, IonToolbar } from '@ionic/react';
import React from 'react';
export function HeaderToolBarWithImage(props: {
    title: React.ReactNode;
    imageName: any;
}) {
    return (<IonToolbar>
        <IonTitle><h1>{props.title}</h1></IonTitle>
        <IonIcon icon={props.imageName} slot='end' class='sleep-icon-header'></IonIcon>
        <IonButtons slot="start">
            <IonBackButton defaultHref="/home" />
        </IonButtons>
    </IonToolbar>);
}
