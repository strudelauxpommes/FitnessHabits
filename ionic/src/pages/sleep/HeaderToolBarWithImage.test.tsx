import React from 'react';
import {HeaderToolBarWithImage} from './HeaderToolBarWithImage';
import renderer from 'react-test-renderer'
import { moon } from 'ionicons/icons';

test('renders correctly', () => {
    const tree = renderer.create(<HeaderToolBarWithImage title="Sommeil" imageName={moon} />).toJSON();
    expect(tree).toMatchSnapshot();
});