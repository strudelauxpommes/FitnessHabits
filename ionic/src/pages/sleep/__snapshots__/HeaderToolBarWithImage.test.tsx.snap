// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ion-toolbar>
  <ion-title>
    <h1>
      Sommeil
    </h1>
  </ion-title>
  <ion-icon />
  <ion-buttons>
    <ion-back-button
      onClick={[Function]}
    />
  </ion-buttons>
</ion-toolbar>
`;
